Fix issues found during testing / preperation for the virtual season

Flova opened the pull request:
Based on #4 

Changes:
- Don't define topics via the json as this seems like a non-standard way of doing this. Instead reasonable defaults are selected and they can easily be remapped by the user. See IPM for a similar discussion.
- Make the json devices file path a parameter
- Check if the json is at the path and give a helpful error if not
- Fix issue where ms are parsed as nanoseconds
- Fix camera info handling (no latch, timestamp, dimensions directly taken from the image)

SammyRamone reviewed (approved):
looks good
thanks for fixing the bugs and improving the code quality

ijnek commented:
@Flova Would you be able to rebase onto current main?

Flova commented:
I did a merge instead of a rebase, because I was on my phone and it was easier that way.

Flova commented:
Now that the CI is working I need to fix linter stuff ^^

Flova commented:
@SammyRamone and @ijnek I addressed the feedback and fixed some linting. I am not able to merge it on this repo so feel free to merge it after approval.

SammyRamone commented:
> @SammyRamone and @ijnek I addressed the feedback and fixed some linting. I am not able to merge it on this repo so feel free to merge it after approval.

looks fine for me, but I will not merge it without @ijnek okay :sweat_smile:

ijnek commented on CMakeLists.txt:
```diff
@@ -56,6 +56,7 @@ install(TARGETS hlvs_player
   DESTINATION lib/${PROJECT_NAME})
 
 install(DIRECTORY launch DESTINATION share/${PROJECT_NAME})
+install(DIRECTORY resources DESTINATION share/${PROJECT_NAME})
```
```suggestion
install(DIRECTORY launch resources
  DESTINATION share/${PROJECT_NAME})
```

ijnek commented on README.md:
```diff
@@ -52,7 +52,7 @@ Run the node using one of these commands:
 # Replace host and port with appropriate values
 ros2 run hlvs_player hlvs_player --ros-args -p host:="127.0.0.1" -p port:=10001
-ros2 launch hlvs_player hlvs_player.launch
+ros2 launch hlvs_player example.launch
```
There's one more reference to ``hlvs_player.launch`` on line 15 in this file which needs to be updated.

ijnek reviewed (approved):
Once the last comments are addressed, the rest looks good. Thanks @Flova!

Flova commented:
I addressed said issue. So we can merge it after the CI finished. Thanks for your review @ijnek !

Flova commented:
CI finished successfully. Merge? @SammyRamone

SammyRamone commented:
> CI finished successfully. Merge? @SammyRamone

I will let @ijnek do it. I'm not sure if normal merge or rebase is preferred for this repo.

ijnek commented:
Thanks @Flova!
